--- configure.orig	2010-11-27 11:10:52.000000000 +0100
+++ configure	2010-12-17 11:51:20.924320896 +0100
@@ -22796,7 +22796,7 @@
 fi
 # }}}
 
-# $PKG_CONFIG --exists OpenIPMIpthread {{{
+# $PKG_CONFIG --exists OpenIPMI {{{
 with_libopenipmipthread="yes"
 with_libopenipmipthread_cflags=""
 with_libopenipmipthread_libs=""
@@ -22816,12 +22816,12 @@
 
 if test "x$with_libopenipmipthread" = "xyes"
 then
-	{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for libOpenIPMIpthread" >&5
-$as_echo_n "checking for libOpenIPMIpthread... " >&6; }
-	$PKG_CONFIG --exists OpenIPMIpthread 2>/dev/null
+	{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for libOpenIPMI" >&5
+$as_echo_n "checking for libOpenIPMI... " >&6; }
+	$PKG_CONFIG --exists OpenIPMI 2>/dev/null
 	if test "$?" != "0"
 	then
-		with_libopenipmipthread="no ($PKG_CONFIG doesn't know OpenIPMIpthread)"
+		with_libopenipmipthread="no ($PKG_CONFIG doesn't know OpenIPMI)"
 	fi
 	{ $as_echo "$as_me:${as_lineno-$LINENO}: result: $with_libopenipmipthread" >&5
 $as_echo "$with_libopenipmipthread" >&6; }
@@ -22829,15 +22829,15 @@
 
 if test "x$with_libopenipmipthread" = "xyes"
 then
-	{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for libOpenIPMIpthread CFLAGS" >&5
-$as_echo_n "checking for libOpenIPMIpthread CFLAGS... " >&6; }
-	temp_result="`$PKG_CONFIG --cflags OpenIPMIpthread`"
+	{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for libOpenIPMI CFLAGS" >&5
+$as_echo_n "checking for libOpenIPMI CFLAGS... " >&6; }
+	temp_result="`$PKG_CONFIG --cflags OpenIPMI`"
 	if test "$?" = "0"
 	then
 		with_libopenipmipthread_cflags="$temp_result"
 	else
-		with_libopenipmipthread="no ($PKG_CONFIG --cflags OpenIPMIpthread failed)"
-		temp_result="$PKG_CONFIG --cflags OpenIPMIpthread failed"
+		with_libopenipmipthread="no ($PKG_CONFIG --cflags OpenIPMI failed)"
+		temp_result="$PKG_CONFIG --cflags OpenIPMI failed"
 	fi
 	{ $as_echo "$as_me:${as_lineno-$LINENO}: result: $temp_result" >&5
 $as_echo "$temp_result" >&6; }
@@ -22845,15 +22845,15 @@
 
 if test "x$with_libopenipmipthread" = "xyes"
 then
-	{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for libOpenIPMIpthread LDFLAGS" >&5
-$as_echo_n "checking for libOpenIPMIpthread LDFLAGS... " >&6; }
-	temp_result="`$PKG_CONFIG --libs OpenIPMIpthread`"
+	{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for libOpenIPMI LDFLAGS" >&5
+$as_echo_n "checking for libOpenIPMI LDFLAGS... " >&6; }
+	temp_result="`$PKG_CONFIG --libs OpenIPMI`"
 	if test "$?" = "0"
 	then
 		with_libopenipmipthread_ldflags="$temp_result"
 	else
-		with_libopenipmipthread="no ($PKG_CONFIG --libs OpenIPMIpthread failed)"
-		temp_result="$PKG_CONFIG --libs OpenIPMIpthread failed"
+		with_libopenipmipthread="no ($PKG_CONFIG --libs OpenIPMI failed)"
+		temp_result="$PKG_CONFIG --libs OpenIPMI failed"
 	fi
 	{ $as_echo "$as_me:${as_lineno-$LINENO}: result: $temp_result" >&5
 $as_echo "$temp_result" >&6; }
